pre-commit

```diff
@@ -101,6 +101,9 @@ check "Web tests" \
 check "Concurrency tests" \
     cargo test -p ipa-core --release --features "shuttle multi-threading"
 
+check "Encrypted Input Tests" \
+    cargo test --test encrypted_input --features "cli test-fixture web-app in-memory-infra"
```

Comment:
@andyleiserson this runs this specific test file, and maybe that's ok. leaving this out just reruns all ~700 tests which we certainly want to avoid. I couldn't figure out how to just run things in the `ipa-core/tests` directory. Is this an OK approach?

Reply:
This seems fine to me.